odc/algo/_memsink.py

```diff
@@ -374,7 +374,7 @@ def da_yxbt_sink(
     return _da_from_mem(token_done, shape=shape, dtype=dtype, chunks=chunks, name=name)
 
 
-def yxbt_sink(ds: xr.Dataset, chunks: Tuple[int, int, int, int]) -> xr.DataArray:
+def yxbt_sink(ds: xr.Dataset, chunks: Tuple[int, int, int, int], name="yxbt") -> xr.DataArray:
```

Comment:
The docstring for this and `yxt_sink` should be updated to include the addition of the `name` parameter.

Reply:
docstring added.